exercises/practice/fizz-buzz/zcl_fizz_buzz.clas.abap

```diff
@@ -0,0 +1,21 @@
+class ZCL_FIZZ_BUZZ definition
+  public
+  final
+  create public .
+
+public section.
```

Comment:
please pretty print this class, keywords upper case

Reply:
plus the testclasses

Reply:
done